reference/docs-conceptual/components/ise/Introducing-the-Windows-PowerShell-ISE.md

```diff
@@ -29,8 +36,8 @@ Key features in Windows PowerShell ISE include:
   SHIFT+ENTER.
```

Comment:
Use the `<kbd>` tags for Shift+Enter, F1, and F5.

Reply:
Fixed all key press instances